tests/upgrades/test_satellitesync.py

```diff
@@ -14,138 +14,120 @@
-        )
-        importing_cvv = self.importing_cv.read().version
-        self.assertTrue(len(importing_cvv) == 1)
-        imported_packages = Package.list({'content-view-version-id': importing_cvv[0].id})
-        self.assertTrue(len(imported_packages) > 0)
-        self.assertEqual(len(exported_packages), len(imported_packages))
-        self.tearDownScenario()
+    org = entities.Organization().search(query={'search': f'name="{pre_test_name}_org"'})[0]
+    request.addfinalizer(org.delete)
+    product = entities.Product(organization=org).search(
+        query={'search': f'name="{pre_test_name}_prod"'}
+    )[0]
+    request.addfinalizer(product.delete)
+    exporting_cv = entities.ContentView(organization=org).search(
+        query={'search': f'name="{pre_test_name}_cv"'}
+    )[0]
+    request.addfinalizer(exporting_cv.delete)
+
+    exporting_cvv_id = max([cvv.id for cvv in exporting_cv.version])
+    exporting_cvv_version = entities.ContentViewVersion(id=exporting_cvv_id).read().version
+
+    ContentView.version_export({'export-dir': f'{export_base}', 'id': exporting_cvv_id})
+    exported_tar = f'{export_base}/export-{exporting_cv.name}-{exporting_cvv_version}.tar'
+
+    result = ssh.command(f"[ -f {exported_tar} ]")
+    assert result.return_code == 0
+
+    exported_packages = Package.list({'content-view-version-id': exporting_cvv_id})
+    assert len(exported_packages) > 0
+
+    importing_org = entities.Organization().create()
+    request.addfinalizer(importing_org.delete)
+    importing_prod = entities.Product(
+        organization=importing_org, name=pre_test_name + "_prod"
+    ).create()
+    request.addfinalizer(importing_prod.delete)
+    importing_repo = entities.Repository(
+        name=pre_test_name + "_repo",
+        mirror_on_sync='no',
+        download_policy='immediate',
+        product=importing_prod,
+    ).create()
+    importing_cv = entities.ContentView(
+        name=pre_test_name + "_cv", organization=importing_org
+    ).create()
+    request.addfinalizer(importing_cv.delete)
+    importing_cv.repository = [importing_repo]
+    importing_cv.update(['repository'])
+    ContentView.version_import({'export-tar': exported_tar, 'organization-id': importing_org.id})
+    importing_cvv = importing_cv.read().version
+    assert len(importing_cvv) == 1
+    imported_packages = Package.list({'content-view-version-id': importing_cvv[0].id})
+    assert len(imported_packages) > 0
+    assert len(exported_packages) == len(imported_packages)
+    ssh.command(f'rm -rf {export_base}/*')
+    exporting_cv_json = exporting_cv.read_json()
+    importing_cv_json = importing_cv.read_json()
+    exporting_cv_env_id = exporting_cv_json['environments'][0]['id']
+    importing_cv_env_id = importing_cv_json['environments'][0]['id']
+    exporting_cv.delete_from_environment(exporting_cv_env_id)
+    importing_cv.delete_from_environment(importing_cv_env_id)
```

Comment:
Should we add some asserts to check if the delete actions were successful?

Reply:
This is not for assertions basically, its a tear-down.

But I am ok even if we assert there.

Reply:
fixed